coretex/entities/task_run/utils.py

```diff
@@ -90,20 +90,28 @@ def getDefaultEntryPoint() -> Optional[str]:
     return None
 
 
+def chunks(lst: list, n: int) -> list:
+    for i in range(0, len(lst), n):
+        yield lst[i:i + n]
```

Comment:
What happens with remainder if `len(lst) % n != 0`?

Reply:
<img width="205" alt="Screenshot 2023-12-08 at 16 51 47" src="https://github.com/coretex-ai/coretexpylib/assets/109690540/5e3f10ef-e948-4054-bc42-0b2d0e67faac">
The remainder gets returned as a list of length len(lst) % n